test/helper.rb

```diff
@@ -149,6 +149,11 @@ def md.directory?
       # Test DH parameters, 2048 bit long safe prime, generator 2 for dh_params in PEM format
       'dh_params.dh_pem' => mockfile.call('dh_params.dh_pem'),
       'default.toml' => mockfile.call('default.toml'),
+      '/etc/postgresql/9.5/main/pg_ident.conf' => mockfile.call('pg_ident.conf'),
+      'C:/etc/postgresql/9.5/main/pg_ident.conf' => mockfile.call('pg_ident.conf'),
+      '/etc/postgresql/9.5/main' => mockfile.call('9.5.main'),
```

Comment:
Also, since I can't add a review comment on an empty file, the mock files:

 * `test/unit/mock/files/var.9.5.main/.gitkeep`
 * `test/unit/mock/files/9.5.main/.gitkeep`

... shouldn't be necessary either. Let's fix whatever is causing you to add these to this PR.